src/MiniValidation/MiniValidator.cs

```diff
@@ -380,7 +380,12 @@ private static async Task<(bool IsValid, IDictionary<string, string[]> Errors)>
 
         foreach (var property in typeProperties)
         {
-            var propertyValue = property.GetValue(target);
+            object? propertyValue = null;
+            try
+            {
+                propertyValue = property.GetValue(target);
+            }
+            catch (Exception) { }
             var propertyValueType = propertyValue?.GetType();
```

Comment:
Shouldn't this fail validation?

Reply:
I don't think so? Certainly the bug report indicates they want the validation to continue, the issue is that for anything that has a getter that throws when in certain state, the walk will cause it to throw. I still feel like I'm missing something here. I might have to try out this scenario in MVC and step through what it does.

Reply:
Should be an opt in. I think blindly swallowing exceptions is bad. Maybe instead have a way to skip a property? (attribute?)

Reply:
Already have a way to opt-out of recursing at the property level: https://github.com/DamianEdwards/MiniValidation/blob/main/src/MiniValidation/SkipRecursionAttribute.cs